Realm/RLMSyncPermissionResults.h

```diff
@@ -0,0 +1,106 @@
+////////////////////////////////////////////////////////////////////////////
+//
+// Copyright 2017 Realm Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+////////////////////////////////////////////////////////////////////////////
+
+#import <Foundation/Foundation.h>
+
+#import "RLMSyncUser.h"
+#import "RLMRealm.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+/// Properties which a sync permission results collection can be sorted by.
+typedef enum : NSUInteger {
+    /// Sort by the path of the Realm the permission pertains to.
+    RLMSyncPermissionResultsSortPropertyPath,
+    /// Sort by the user ID of the user the permission applies to.
+    RLMSyncPermissionResultsSortPropertyUserID,
+} RLMSyncPermissionResultsSortProperty;
+
+@class RLMSyncPermissionValue;
+
+/**
+ An object representing the results of a permissions query.
```

Comment:
We should state that `RLMSyncPermissionResults` is an auto-updating collection but its values are static.